frontend/ui/lib/pages/trip_view_page.dart

```diff
@@ -1,13 +1,19 @@
 import 'package:flutter/material.dart';
+import 'package:tripmeout/services/trip_service.dart';
 import 'package:tripmeout/widgets/trip_view_widget.dart';
 import 'package:tripmeout/widgets/default_app_bar.dart';
 
 class TripViewPage extends StatelessWidget {
+  final TripService tripService;
+  final String tripId;
+
+  TripViewPage(this.tripService, this.tripId, {Key key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: defaultAppBar(context),
-      body: Center(child: TripViewWidget()),
+      body: Center(child: TripViewWidget(this.tripService, this.tripId)),
```

Comment:
Is it necessary to keep the `Center` here? The answer may be yes, I just want to make sure that each widget has a purpose.

Reply:
I can't even remember I think this use to matter but with how the new trip view page is now I don't think that will matter anymore.